anyway/parsers/infographics_data_cache_updater.py

```diff
@@ -293,7 +293,7 @@ def get_cache_info():
         db.session.query(NewsFlash)
         .filter(NewsFlash.accident)
         .filter(NewsFlash.resolution.in_(["כביש בינעירוני"]))
-        .filter(not_(NewsFlash.road_segment_name == None))
+        .filter(not_(NewsFlash.road_segment_name is None))
```

Comment:
Notice it's not a pythonic operation but SqlAlchemy, from reading this message on Stackoverflow(below) i don't believe it will work

https://stackoverflow.com/a/5632224/13771969